packages/bot/src/components/saveload-modal.jsx

```diff
@@ -0,0 +1,197 @@
+                                                {translate('A collection allow you to save block as an individual part which can be add into other bot')}
+                                            </div>
+                                        </>
+                                        :   <input
+                                            type='file'
+                                            id='files'
+                                            style={{ display: 'none' }}
+                                            onChange={handleFileChange}
+                                        />
+                                }
+                            </div>
+                            <div className='modal__footer'>
+                                <Button
+                                    type='button'
+                                    className={classNames(
+                                        'save-load__button',
+                                        'btn--secondary',
+                                        'btn--secondary--orange',
+                                    )}
+                                    text={translate('Cancel')}
+                                    onClick={() => toggleSaveLoadModal(is_save_modal)}
+                                />
+                                <Button
+                                    type='submit'
+                                    className={classNames(
+                                        'save-load__button',
+                                        'btn--primary',
+                                        'btn--primary--orange',
+                                    )}
+                                    is_loading={button_status === 1}
+                                    is_submit_success={button_status === 2}
+                                    text={translate('Continue')}
+                                />
+                            </div>
+                        </Form>
+                    )
+                }
+            </Formik>
+        </Modal>
+    );
+};
+
+const IconRadio = ({
+    icon,
+    text,
+    google_drive_connected,
+    onDriveConnect,
+}) => {
+    const is_drive_radio = text === 'Google Drive';
+
+    return (
+        <div className='saveload-type__container'>
+            <div className='saveload-type__radio'>
+                {
+                    icon &&
+                    React.cloneElement(
+                        icon,
+                        { className: classNames('saveload-type__icon', icon.props.className) },
+                    )
+                }
+                <p className='saveload-type__text'>{translate(text)}</p>
+            </div>
+            {
+                is_drive_radio &&
+                <p className='saveload-type__drive-status' onClick={onDriveConnect}>{google_drive_connected ? 'Disconnect' : 'Connect'}</p>
```

Comment:
Make sure to translate when required.

```suggestion
                <p className='saveload-type__drive-status' onClick={onDriveConnect}>
                    {google_drive_connected ? translate('Disconnect') : translate('Connect')}
                </p>
```